### ISSUE TITLE
Task: "Enabling Ingress Traffic", secure ingress times out

### ISSUE THREAD

**Issue by ahmetb** [istio/istio.github.io#160]
https://istio.io/docs/tasks/ingress.html#configuring-secure-ingress-https

Steps followed:
- installed httpbin sample, it worked
- applied Configuring ingress (HTTP), worked
- applied Configuring secure ingress

Now `curl http://$INGRESS_URL/headers` works but `curl -k https://$INGRESS_URL/html` is giving timeout on curl:

```
$ kubectl describe ingress

Name:			istio-ingress
Namespace:		default
Address:		35.185.252.35
Default backend:	default-http-backend:80 (10.4.3.2:8080)
Rules:
  Host	Path	Backends
  ----	----	--------
  *
    	/headers 	httpbin:8000 (<none>)
    	/delay/.* 	httpbin:8000 (<none>)
Annotations:
Events:	<none>

----------------------------------------------------
Name:			secured-ingress
Namespace:		default
Address:		35.185.252.35
Default backend:	default-http-backend:80 (10.4.3.2:8080)
TLS:
  ingress-secret terminates
Rules:
  Host	Path	Backends
  ----	----	--------
  *
    	/html 	httpbin:8000 (<none>)
Annotations:
Events:	<none>
```

**Comment by PiotrSikora** [comment]
`curl http://A.B.C.D/headers` (default port 80)
`curl -k https://A.B.C.D/html` (default port 443)

If `$INGRESS_URL` is set to `A.B.C.D:80` then the `https://` case will try to connect on port 80 and not 443.

**Comment by frankbu** [comment]
Fixed in #176
